packages/nightingale-sequence/src/nightingale-sequence.ts

```diff
@@ -193,7 +187,12 @@ class NightingaleSequence extends withManager(
 
       // only add axis if there is room
       if (this.height > (this.chWidth || 0) && this.xScale) {
-        const xAxis = axisBottom(this.xScale)
+        // Copying the scale, and rounding its domain values to avoid bug releated with floats
+        const roundScale = this.xScale.copy();
```

Comment:
Hey @Two-Kay 
I added the `.copy()` here to make sure changes in `roundScale` won't affect `xScale`